neural_tangents/stax.py

```diff
@@ -3775,7 +3778,7 @@ def _mean_and_var(
     return x, var
 
   if mask is None:
-    mean = np.mean(x, axis, dtype, out, keepdims)
+    mean = np.mean(x, axis, dtype, keepdims)
     if get_var:
       var = np.var(x, axis, dtype, out, ddof, keepdims)
```

Comment:
I noticed you removed `out` from `mean` arguments, but not here - should it be removed from here (and perhaps from the `_mean_and_var` at all)?

Reply:
Roman, thanks for pointing this out! I removed the `out` argument in `np.var` in the latest commit. Maybe it is better to keep the `_mean_and_var` interface as its origin form (in order to avoid many changes in the places where `_mean_and_var` are called)?

Reply:
I think it's not being used anywhere (and perhaps shouldn't have been introduced in the first place), so I would just remove it everywhere, but I'm ok either way here

Reply:
I see. Thanks!